src/panel/pages/timeline/Timeline.tsx

```diff
@@ -64,6 +64,9 @@ const TimelineList = styled.div`
   flex-direction: column;
   position: relative;
   margin: 40px 0;
+  &:active {
+    cursor: grabbing;
```

Comment:
I was hoping we wouldn't have to do some hacky stuff to get grabbing to work.

Never would have guessed this would work!!!

Reply:
Tell me about it! Much easier than injecting the css from the handler 😅